docs/Using-Fleet/REST-API.md

```diff
@@ -5170,6 +5172,12 @@ Returns a list of all queries in the Fleet instance.
     "name": "query1",
     "description": "query",
     "query": "SELECT * FROM osquery_info",
+    "team_id": null,
```

Comment:
Needs to include a true/false for automation.

Reply:
For the query YAML we decided to nest snapshot and ignore removals under automations, reckon it'd make sense to go that route here instead of a true/false?
![Screenshot 2023-06-06 at 4 04 39 PM](https://github.com/fleetdm/fleet/assets/3065949/44f03ed2-b5de-4ddb-acca-b44b97ddd7cb)

Reply:
either have a blank object here or have it with the items filled in.